4.interpret_model/scripts/nbconverted/interpret_model.py

```diff
@@ -86,7 +103,7 @@
 ax = sns.barplot(data=pheno_class_ordered)
 
 
-# In[8]:
+# In[9]:
```

Comment:
The chart generated with this cell seemed quite large and possibly unreadable (depending on how it is rendered). While it didn't appear to change with this PR I wanted to call it out just in case it's helpful.

Reply:
This was a prototype chart that may be removed in the future. I think for now we should keep it as we haven't quite decided how to interpret many of the charts in this module and it could be useful in the future.